DATA/MISSIONS/npcships.ini

```diff
@@ -54,6 +144,51 @@ state_graph = TRANSPORT
 pilot = transport_easy
 npc_class = lawful, class_transport, transport_large, d1, d2, d3, d4, d5, d6, d7, d8, d9, d10, d11, d12, d13, d14, d15, d16, d17, d18, d19
 
+[NPCShipArch]
+nickname = npc_co_br_large_transport_d1-19
+level = d10
+loadout = co_br_large_transport_loadout_d1-19
+ship_archetype = cv_ship_transport03_npc
+state_graph = TRANSPORT
+pilot = transport_easy

```

Comment:
I wonder if we shouldn't give transport_ace to all the transports. Small, med, large, etc.

Reply:
will be changed in future NPC Adjustment phases